bugwarrior/services/__init__.py

```diff
@@ -368,6 +368,8 @@ def build_default_description(
             'release': 'Release',
             'chore': 'Chore',
         }
+        if cls not in cls_markup.keys():
+            cls_markup[cls] = cls.lower()
```

Comment:
It's not clear to me why this is necessary since `task` and `bug` are both keys in `cls_markup`.

Reply:
Task and Bug aren't the only workitem types that can be assigned to users in Azure Devops. It supports different methodologies, which call tracked tasks different things that aren't included in the dict, like `product backlog item` and `impediment` for scrum process and `change request` and `risk` for CMMI.

Reply:
In the past we've added items to `cls_markup` for each individual type, even if it's only used by a single service. (See #666 in which we added 5 items for one service.) It would probably be better to move this logic down into the individual services but I don't think we'll want to accept open-ended types like this.

Reply:
Perhaps I don't understand this portion of code like I thought I did. I was under the impression that all this provides is a work item type in the default description, and gives it some shorthand or removed it in special cases. I'd be fine doing it either way, and even trying to abstract this into the config somehow, but I don't understand why open ended types would be an issue.

Reply:
You're right. In general we prefer that bugwarrior breaks when our API assumptions are violated but in this case there don't appear to be consequences for arbitrary classes.

However, for consistency could we make this  `cls.title()`? You could then remove the last 5 items in `cls_markup` which would be effectively no-op.

Also, I think it would be more straightforward to do `cls_markup.get(cls, cls.title())` in the returned string below than to add it to the dict like this.